src/test/scala/is/hail/methods/AggregatorSuite.scala

```diff
@@ -390,4 +390,81 @@ class AggregatorSuite extends SparkSuite {
     } (Parameters(rng, 1000, 100))
   }
 
+  private def runAggregatorExpression(expr: String, aggregableName: String, aggregableElementType: Type, aggregableValue: TraversableOnce[_]): Any = {
```

Comment:
This is nice.  I think we could extend HailContext.eval() (or a variant) to take a set of bindings in which to evaluate the expression, including aggregables.  That would be super useful.